iquip/apps/scheduler.py

```diff
@@ -209,6 +205,8 @@ def paint(self,
         index: QModelIndex):
         """Overridden."""
         data = index.data(Qt.DisplayRole)
+        if data is None:
```

Comment:
I added this because when `_syncExperimentQueue` is called and the input value is `experimentList = []`, the `self.schedulerFrame.model`'s `dataChanged` call gives an error, as there is nothing to refresh. Here, adding the case when `data` is `None` enables it to think there is something (which is nothing as you can see: `QSize(0, 0)`) to refresh.